apps/demos/Demos/Chat/Customization/jQuery/styles.css

```diff
@@ -0,0 +1,52 @@
+.dx-viewport {
+    padding: 20px;
+    min-width: 780px;
+    border: 1px solid var(--dx-color-border);
+    border-radius: 8px;
+}
+
+.demo-container {
+    min-width: 720px;
+    display: flex;
+    gap: 20px;
+}
+
+.chat-container {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    align-items: center;
+    justify-content: center;
+}
+
+.options {
+    padding: 20px;
+    min-width: 280px;
+    background-color: rgba(191, 191, 191, 0.15);
+}
+
+#chat {
+max-width: 480px;
+}
+
+.caption {
+    font-size: 18px;
+    font-weight: 500;
+}
+
+.option-group {
+    padding: 10px 0;
```

Comment:
instead of this use `padding-block` with correspond value, check margin too. Also your option-group can be flex container, in this case you can use 'gap' property instead on margin

Reply:
https://github.com/DevExpress/DevExtreme/pull/28441/commits/7abeedba5198baaf75acd81fdb9e03e779b24d94